tests/snapshots/tests.contrib.sanic.test_sanic_server.test_sanic_errors.json

```diff
@@ -0,0 +1,59 @@
+[[
+  {
+    "name": "sanic.request",
+    "service": "sanic",
+    "resource": "sanic.request",
+    "trace_id": 0,
+    "span_id": 1,
+    "parent_id": 0,
+    "type": "web",
+    "error": 1,
```

Comment:
This looks like a regression? Error should not be set for a 404 and we probably don't want the exception info either.

The previous test case for this asserted that error was `0`.